### PR TITLE
feat: add links to offchain::ipfs and http tutorial

### LINKED ISSUE #10: IA: Where to put a couple links?
We have, or will soon have, links now for:
- https://rs-ipfs.github.io/offchain-ipfs-manual/
- https://github.com/rs-ipfs/rust-ipfs/pull/420

How can we fit these onto the site in a cohesive way? Open to any and all suggestions.

### PR DESCRIPTION
**Pull request by niklaslong**
First quick prototyping, still missing a link for the http tutorial (will be added once merged). Adjusted the link colour to a slightly darker hue for contrast on the green. 

Closes #10.

Screenshot 2020-10-30 at 16 43 46

### CODE REVIEW

**Comment by aphelionz** [comment]
Awesome, can you include a screenshot in the PR description?

**Comment by niklaslong** [comment]
I also (very quickly) played around with the colours on another branch, not sure if this is open for discussion (if it isn't feel free to ignore this 😉) but thought I'd share:

Screenshot 2020-10-30 at 17 28 33

File: index.html [hunk 1]
```diff
@@ -169,6 +169,14 @@ <h2>Getting Started</h2>
       <a class="top" href="#">▲</a>
     </section>
 
+    <section id="further-reading">
+      <h2>Further Reading</h2>
+      <p>These are some extra resources to get you started with specific use-cases of Rust IPFS:</p>
+      <p><a href="https://rs-ipfs.github.io/offchain-ipfs-manual/">The offchain::ipfs manual</a>: a project integrating IPFS and Substrate.</p>
```
Comment (aphelionz) [inline index.html]: Had you considered an `<ol>` here?

**Review by aphelionz — APPROVED** [review]
LGTM, I think we can merge as-is. Just one very quick question.

File: index.html (hunk 1 above)
Comment (niklaslong) [reply index.html]: I'd considered an `<ul>`, might be better to be honest 🤔

Comment (niklaslong) [reply index.html]: Screenshot 2020-10-30 at 18 54 33

**Comment by niklaslong** [comment]
I've added the future link to the getting started guide, we'll need to merge https://github.com/rs-ipfs/rust-ipfs/pull/420 as well. I've also changed to a `ul`.

**Comment by aphelionz** [comment]
Oh! One more thing - either move that accessibility link "back to top" to under the further reading section, or add a new one there

**Comment by niklaslong** [comment]
Ah, good catch—done ✅

File: index.html [hunk 2]
```diff
@@ -169,6 +169,19 @@ <h2>Getting Started</h2>
       <a class="top" href="#">▲</a>
     </section>
 
+    <section id="further-reading">
+      <h2>Further Reading</h2>
+      <p>These are some extra resources to get you started with specific use-cases of Rust IPFS:</p>
+      <div>
+        <ul>
+          <li><a href="https://rs-ipfs.github.io/offchain-ipfs-manual/">The offchain::ipfs manual</a>: a project integrating IPFS and Substrate.</li>
+          <!-- TODO: Add link once merged -->
```
Comment (aphelionz) [inline index.html]: Remove this comment and we'll me good to merge!

**Review by aphelionz — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (niklaslong) [reply index.html]: Fixed 👍

**Review by aphelionz — APPROVED** [review]

**Pull request merged**